viam-cartographer.go

```diff
@@ -325,6 +341,13 @@ func (cartoSvc *cartographerService) getNextDataPoint(ctx context.Context, lidar
 	}
 }
 
+// Reconfigure always returns a must rebuild error and requires a map update, so the
+// map timestamp is updated.
+func (cartoSvc *cartographerService) Reconfigure(ctx context.Context, deps resource.Dependencies, conf resource.Config) error {
+	cartoSvc.mapTimestamp = time.Now()
```

Comment:
[q] did you look at all into the behavior of reconfigure and if a return of `resource.NewMustRebuildError` will call New again?

Reply:
Nope will do that

Reply:
As far as I can tell, the error only raises an error and does nothing else (i.e. no calls to New). It seems the expectation is that models implement their own versions of this error, and are advised they "should be able to reconfigure in place to support the best user experience" but this is not currently being done in cartographer.